src/metabase/util/malli.clj

```diff
@@ -12,17 +12,23 @@
    [metabase.util :as u]
    [ring.util.codec :as codec]))
 
+(core/defn- nil->nil-str [x]
+  (if (nil? x) "nil" x))
+
 (core/defn- ->malli-io-link
   ([schema]
-   (->malli-io-link schema (try (mg/generate schema {:seed 1 :size 1})
-                                ;; not all schemas can generate values
-                                (catch Exception _ ::none))))
+   (->malli-io-link schema (try
+                             ;; try to make a sample value
+                             (mg/generate schema {:seed 1 :size 1})
+                             ;; not all schemas can generate values
+                             (catch Exception _ ::none))))
   ([schema value]
-   (let [url-schema (codec/url-encode (u/pprint-to-str (mc/form schema)))
-         url-value (if (= ::none value)
-                     ""
-                     (codec/url-encode (u/pprint-to-str value)))]
-     (str "https://malli.io?schema=" url-schema "&value=" url-value))))
+   (str "https://malli.io?schema="
+        (codec/url-encode (u/pprint-to-str (mc/form schema)))
+        "&value="
+        (if (= ::none value)
+          ""
+          (codec/url-encode (nil->nil-str (u/pprint-to-str value)))))))
```

Comment:
I'm very afraid of changing the semantics of something in `metabase.util`, but;

```
(defn pprint-to-str
  "Returns the output of pretty-printing `x` as a string.
  Optionally accepts `color-symb`, which colorizes the output with the corresponding
  function from `colorize.core`.

     (pprint-to-str 'green some-obj)"
  (^String [x]
   (when x
     (with-open [w (java.io.StringWriter.)]
       (pprint x w)
       (str w))))

  (^String [color-symb x]
   (colorize color-symb (pprint-to-str x))))
   ```
   
   I am very surprised it has that `(when x ...)` and tentatively support removing it. For comparison,
   
   ```
   (with-open [w (java.io.StringWriter.)]
       (pprint nil w)
       (str w))
;; =>
"nil\n"
```

and of course

```
(pprint nil)
;; prints 'nil' to stdout
;; returns nil
```